src/main/java/io/jenkins/plugins/report/jtreg/main/comparator/Jobs.java

```diff
@@ -22,16 +22,13 @@ public Jobs(String jobsDirectory) {
     }
 
     // prints all jobs that match the query string
-    public void printJobs(String queryString, boolean showNvrs) {
-        for (int i = 0; i < jobsInDir.length; i++) {
-            if (QueryString.checkJobWithQuery(jobsInDir[i], queryString)) {
-                System.out.printf("%d) %s%n", i, jobsInDir[i].getName());
-                if (showNvrs) {
-                    for (String nvr : Builds.getJobNvrs(jobsInDir[i])) {
-                        System.out.print(nvr + ", ");
-                    }
-                    System.out.print("\n\n");
-                }
+    public void printJobs(String queryString, boolean skipFailed, String nvrQuery, int numberOfBuilds) {
+        ArrayList<File> jobs = getJobsByQuery(queryString);
+        for (File job : jobs) {
+            System.out.println(job.getName());
```

Comment:
I would recomed to to crreate class Outputs, where will be managed stdout/err, with some posisble debug on/of.  it may happen that will contian nol two methods print and printe, with system.out.print and system.err.print but that is enough. It is then muc more easily trackable in tool, and allow snice operation in future.